### PR TITLE
Makes bad compiler args into compiler warnings

### PR DESCRIPTION
**Pull request by ike709**

### CODE REVIEW

**Comment by wixoaGit** [comment]
Why is this preferable? It seems to me that if you make a mistake in the arguments you wouldn't want the compiler to go on.

**Comment by ike709** [comment]
Because I'm tired of needing to remember to remove `--experimental-preproc` from all of my Rider configurations.

**Comment by ike709** [comment]
TODO: Add a flag to cope with bad flags.

File: DMCompiler/Program.cs [hunk 1]
```diff
@@ -16,22 +18,29 @@ class Program {
...
                     default: {
                         string extension = Path.GetExtension(arg);
 
                         if (!String.IsNullOrEmpty(extension) && (extension == ".dme" || extension == ".dm")) {
                             settings.Files.Add(arg);
                             Console.WriteLine($"Compiling {Path.GetFileName(arg)}");
                         } else {
-                            Console.WriteLine($"Invalid arg '{arg}'");
-                            return false;
+                            if(skipBad || args.Contains("--skip-bad-args")) {
```
Comment (wixoaGit) [inline DMCompiler/Program.cs:38]: The `skipBad` variable can be removed if you're just going to end up doing `args.Contains("--skip-bad-args")` anyways.

**Review by wixoaGit — CHANGES_REQUESTED** [review]

File: DMCompiler/Program.cs (hunk 1 above)
Comment (wixoaGit) [reply DMCompiler/Program.cs:38]: That or initialize `skipBad` to be the Contains call

**Review by wixoaGit — APPROVED** [review]

**Pull request merged**